### PR TITLE
Add a pip module files

### PR DESCRIPTION
**Pull request by fonsecareyna82**
Add a pip module files

### CODE REVIEW

File: relion/__init__.py [hunk 1]
```diff
@@ -28,16 +28,15 @@
...
 
 
 _logo = "relion_logo.png"
 
 
 class Plugin(pyworkflow.em.Plugin):
     _homeVar = RELION_HOME
-    _pathVars = [RELION_HOME]
-    _supportedVersions = [V2_0, V2_1]
+    _supportedVersions = [V2_0, V2_1, V1_4, V1_4f, V1_3]
```
Comment (delarosatrevin) [inline relion/__init__.py]: @fonsecareyna82 Versions prior to 2.0 will not be supported any longer

File: relion/__init__.py [hunk 2]
```diff
@@ -65,7 +64,43 @@ def getEnviron(cls):
...
+    @classmethod
+    def defineBinaries(cls, env):
+        relion_commands = [('./INSTALL.sh -j %d' % env.getProcessors(),
+                            ['relion_build.log',
+                             'bin/relion_refine'])]
+
+        env.addPackage('relion', version='1.4',
+                       tar='relion-1.4.tgz',
+                       commands=relion_commands,
+                       default=False)
```
Comment (delarosatrevin) [inline relion/__init__.py]: If default=False the "default" option? If so, remove from here, if not, add to the other binaries except for the last one

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    # https://packaging.python.org/specifications/core-metadata/#name
+    name='scipion-em-relion',  # Required
+
+    # Versions should comply with PEP 440:
+    # https://www.python.org/dev/peps/pep-0440/
+    #
+    # For a discussion on single-sourcing the version across setup.py and the
+    # project code, see
+    # https://packaging.python.org/en/latest/single_source_version.html
+    version='2.1',  # Required
```
Comment (delarosatrevin) [inline setup.py]: I think version of the plugin should not be the 2.1 (the relion one, I guess). It should be 0.0.x, but we can change that.

Comment (fonsecareyna82) [reply setup.py]: Can we eliminate the possibility of installing versions of binaries lower than 2.0?

Comment (delarosatrevin) [reply setup.py]: Now 1.4 is needed by localrec...but we should eliminate them as soon as they are not needed anymore.

Comment (fonsecareyna82) [reply setup.py]: 0K. I will not eliminate the definition of these binaries

**Comment by azazellochg** [comment]
this one. Other one requires some testing :))

**Pull request merged**